src/messages.tsx

```diff
@@ -0,0 +1,20 @@
+export const NO_CONTEXTS_SAVED = 'No contexts saved yet, run asyncapi --help to know more';
+
+export const CONTEXT_NOT_FOUND = 'This context key does not exists.';
+
+export const KEY_NOT_FOUND = 'Key not found';
+
+export const DELETING_CURRENT_CONTEXT = 'You are tyring to delete a context that is currently in use.';
+
+export const MISSING_CURRENT_CONTEXT = 'No context is set as current, please set a current context.';
+
+export const MISSING_ARGUMENTS = 'Missing arguments.';
```

Comment:
I don't think `MissingArgumentstError` is ever used

Reply:
This is used when any input variables are missing 

https://github.com/asyncapi/cli/blob/2f69e066f00c1dcb8336b1f1f2d8aac09d23884b/src/hooks/context/models.ts#L50-L55

Reply:
let's leave it as it is for now